ansible/roles/cursor/tasks/main.yml

```diff
@@ -75,3 +75,88 @@
   become: true
   when: cursor_download is succeeded
   changed_when: true
+
+# Cursor configuration for performance optimization
+- name: Ensure jq is installed for JSON manipulation
+  ansible.builtin.package:
+    name: jq
+    state: present
+  become: true
+
+- name: Create Cursor config directory if it doesn't exist
+  ansible.builtin.file:
+    path: "{{ cursor_config_dir }}"
+    state: directory
+    mode: "0755"
+
+- name: Check if Cursor settings file exists
+  ansible.builtin.stat:
+    path: "{{ cursor_settings_file }}"
+  register: cursor_settings_file_stat
+
+- name: Create empty settings file if it doesn't exist
+  ansible.builtin.copy:
+    content: "{}"
+    dest: "{{ cursor_settings_file }}"
+    mode: "0644"
+  when: not cursor_settings_file_stat.stat.exists
+
+- name: Backup existing Cursor settings
+  ansible.builtin.copy:
+    src: "{{ cursor_settings_file }}"
+    dest: "{{ cursor_settings_file }}.backup.{{ ansible_date_time.epoch }}"
+    mode: "0644"
```

Comment:
There's no cleanup of backups included.  This will be unreasonable over time.  Backup is not a requirement.